### ISSUE TITLE
WebGL2 support

### ISSUE THREAD

**Issue by Trass3r** [spite/ShaderEditorExtension#20]
Is it supported?
http://toji.github.io/webgl2-particles/
When clicking on reload in the shader editor no gui appears with Vivaldi (Chrome 56).

**Comment by spite** [comment]
No, WebGL2RenderingContext is not yet supported. Coming up in a revamped version of the editor.

**Comment by leo-cheron** [comment]
Any news of that revamped version, 5 years later? 👯

**Comment by makskv** [comment]
Still waiting for this banger
